stub/src/main/java/io/grpc/stub/CallStreamObserver.java

```diff
@@ -50,7 +48,6 @@
  * <p>DO NOT MOCK: The API is too complex to reliably mock. Use InProcessChannelBuilder to create
  * "real" RPCs suitable for testing.
  */
-@ExperimentalApi("https://github.com/grpc/grpc-java/issues/1788")
 public abstract class CallStreamObserver<V> implements StreamObserver<V> {
```

Comment:
I thought this class was to remain experimental.

Reply:
If this class remains experimental, then its subclasses should also be experimental. We can not have a stable class extend an experimental class.

Reply:
We talked of redefining the methods in the subclasses, so the only issue is if someone references CallStreamObserver directly.

Reply:
Reverted `CallStreamObserver.java`.

Reply:
Actually I don't recall the details and don't find anything specific in the experimental API meeting notes or the issue #1788 . @ejona86 What else is needed?

Reply:
I was referencing https://github.com/grpc/grpc-java/issues/1788#issuecomment-453227259 before.

Reply:
Redefined methods of `CallStreamObserver` in subclasses.